pyproject.toml

```diff
@@ -120,6 +120,7 @@ omit = [
     "manage.py",
     "alexandria/wsgi.py",
     "generate_missing_thumbnails.py",
+    "generate_content_vectors.py",
```

Comment:
I'd prefer to have this tested, except the memory bit...

Reply:
Added the negative test